README.rst

```diff
@@ -660,6 +660,27 @@ Example::
 Response::
     Binary file
 
+@@send-booking-reminders
+------------------------
+
+This view sends a bookign reminder email to all the bookings inside the PrenotazioniFolders which have the popolated Reminder Notification Gap field.
+If you mind to setup a cronjob to call this view, you migth use a special script call. The script is located at src/redturtle/prenotazioni/scripts/notify_upcomming_bookings.pyp
+
+
+Scripts
+=======
+
+notify_upcomming_bookings
+-------------------------
+
+The script is might be used to call the **@@send-booking-reminders view**.
```

Comment:
probably it's better "the script can be used"

Reply:
"It is supposed" i think

Reply:
Solved